### ISSUE TITLE
Terminology change

### ISSUE THREAD

**Issue by scottrCT** [airsalliance/lov#66]
Suggesting a terminology change: Change all the "exclude" attributes to "include". This makes coding more logical as the true/false values line up with what you are really thinking/checking for (i.e. do I include this in the website or not).

**Comment by klambacher** [comment]
I disagree with this one - exclusions are rare vs inclusions. Most existing systems already work on an exclusion basis primarily. We have both a non-public boolean setting for records, and a record inclusion list for specific directories. They are both useful, but in this context I think the exclude from website setting would be akin to our non-public field and would prefer it remain an exclusion field.

**Comment by eric-jahn** [comment]
I guess it gets into policy of an I&R as far as whether there is a default for exclusion.  I could see an I&R needing it to default one way or the other, depending on their typical usage.  Like, does the lack of a positive "exclude=true" directive imply it should be included?  I think it should always be explicit to avoid misuse.  Luckily, it's a required field in the XML, so it won't be ambiguous when transmitted, but it is not required in the vocabulary (which is a bug; they should be the same).  I personally feel "exclude" = don't put on website, is clear enough, but I do think it needs to be clarified on a policy/storage/transmission level.  Probably something more fine grained that public /not public is in order as well, long-term.  Like, does this mean, in all cases, to exclude from a public website, or just any html at any level of authorization?  I'm sure there is guidance on this somewhere, as far as conventions for use, if someone can dig it up.

**Comment by scottrCT** [comment]
I want to point out I am suggesting this purely from a consumer of the data having to process the file. When processing the file, you have to process that field regardless of the value. To have it be an exclusion rather than an inclusion is just essentially syntax. I like to call these "negative" fields, meaning when processing, you would have to understand the boolean value as the opposite of what it is intending (i.e. do not show the field is true, show the field is false). Just wanted to throw the suggestion out there and am appreciating the points of view and thoughts on the methodology behind it.
